v1/pipeline/steps/custom/key-values-criteria-spec.yaml

```diff
@@ -9,11 +9,8 @@ allOf:
       type: array
       items:
         $ref: condition.yaml
-    matchAnyCondition:
+    match:
```

Comment:
can we make this field mandatory or with a default value, if used in future in V1.
It is required here: io/harness/steps/approval/step/beans/KeyValuesCriteriaSpecDTO.java:53. Currently FE always populates it

Reply:
this class is not being used anywhere

Reply:
yeah adding it basically prevents reviewing this file again incase we need to add it in future